Boards.h

```diff
@@ -276,6 +276,22 @@ writePort(port, value, bitmask):  Write an 8 bit port.
 #define PIN_TO_PWM(p)           PIN_TO_DIGITAL(p)
 #define PIN_TO_SERVO(p)         (p) // deprecated since v2.4
 
+// Arduino MKRFox1200
+#elif defined(ARDUINO_SAMD_MKRFox1200)
+#define TOTAL_ANALOG_PINS       7
+#define TOTAL_PINS              33 // 8 digital + 3 spi + 2 i2c + 2 uart + 7 analog + 3 usb + 1 aref + 5 sd + 1 bottom pad + 1 battery adc
+#define IS_PIN_DIGITAL(p)       ((((p) >= 0 && (p) <= 21)) && !IS_PIN_SERIAL(p))
```

Comment:
But is there any reason these pins cannot be used for digital I/O? It has not been an issue with other boards (since `Serial1.begin()` is only called if the user sends a SERIAL_CONFIG message). Only pins tied to `Serial` cannot be used for digital I/O on other boards.